src/components/o-cookie-disclaimer/_template.js

```diff
@@ -0,0 +1,23 @@
+import html from 'nanohtml';
+
+export default ({
+  classes,
+  buttonName,
+  title,
+  linkTitle,
+  href,
+  fixed = false,
+}, childrenFragment) => html`
+  <article class="${classes} ${fixed ? 'o-cookie-disclaimer--fixed' : ''}">
+    <axa-container>
```

Comment:
i know here that freedom is not there in using childrens, but i thought is better/easier for the user of this to not have to worry to pass too many child components

Reply:
You could also just use the container mixins and adress this with sass